stun_server/main.tf

```diff
@@ -0,0 +1,40 @@
+terraform {
+  cloud {
+    organization = "home_assistant"
+
+    workspaces {
+      name = "stun_server"
+    }
+  }
+}
+
+provider "aws" {
+  region = "us-east-1"
+}
+
+module "us_east_1" {
+  source = "./region"
```

Comment:
module is missing aws config, so us-east-1 will be used.

Reply:
Added.